### PR TITLE
Update accessibility-quiz.ts

### PR DESCRIPTION
**Pull request by AaadityaG**
## Summary of changes

fixing #374 

I added three questions and answers : 
1. When was version 1.0 of the Web Content Accessibility Guidelines (WCAG) released?
2. Which of the following is NOT the way to fix web accessibility errors?
3. What is web accessibility?

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/accessibility-quiz.ts
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:542]: question already exists in the quiz file
please remove

File: src/data/accessibility-quiz.ts
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:550]: fixing spelling errors and grammar issues 

```suggestion
      "Fixing empty links, adding missing alt text for images and updating low contrast text are some common ways to fix the accessibility errors",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/accessibility-quiz.ts
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:540]: This still needs to be fixed.
There is a lot of words in this sentence that don't need to be capitalized.
For example, "Fix Empty Links" should just be "Fixing empty links"
Same with "Adding Missing Alternative Text for Images" can just be "adding missing alternative text for images"

please refer to my previous review comment on the fixes for the capitalization 👍

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

File: src/data/accessibility-quiz.ts
Comment (jdwilkin4) [reply src/data/accessibility-quiz.ts:540]: @AaadityaG 

Here is the previous review comment with the correct changes 

https://github.com/freeCodeCamp/Developer_Quiz_Site/pull/791#discussion_r1355700533

hope that helps

Comment (AaadityaG) [reply src/data/accessibility-quiz.ts]: done! Can I hope (hacktoberfest) label?

**Comment by AaadityaG** [comment]
I hope I am done now...😅
Can I get hacktoberfest label

**Comment by jdwilkin4** [comment]
> Can I get hacktoberfest label

Yes, I will go ahead and approve it and once it gets merged it will be accepted for hacktoberfest.
That applies to all repos participating in hacktoberfest 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**
